src/gha_runner/__main__.py

```diff
@@ -142,6 +146,12 @@ def main():  # pragma: no cover
     provider = os.environ.get("INPUT_PROVIDER")
     if provider is None:
         raise Exception("Missing required input variable INPUT_PROVIDER")
+    gh_timeout_var = os.environ.get("INPUT_GH_TIMEOUT")
+    # Set the default timeout to 10 minutes
+    gh_timeout = 1200
+    # If the user has set the timeout, we will use that instead
+    if gh_timeout_var is not None:
```

Comment:
Checking on the logic here: does this show up as `None` (not in the environ dict) or does it show up as an empty string (in the environ dict, but not set to a meaningful value)?

If it is actually `None`, wouldn't the following be an easier way to extract or get default?

```python
gh_timeout = int(os.environ.get("INPUT_GH_TIMEOUT", 1200))
```